minSDKversion - FAILURE: Build failed with an exception.

### What
FAILURE: Build failed with an exception.

* Where:
Build file '/home/runner/work/smooth-app/smooth-app/packages/smooth_app/android/app/build.gradle.kts' line: 26

* What went wrong:
Script compilation errors:

  Line 26:         minSdkVersion flutter.minSdkVersion
                                        ^ Expecting an element

  Line 26:         minSdkVersion flutter.minSdkVersion
                   ^ Function invocation 'minSdkVersion(...)' expected

  Line 26:         minSdkVersion flutter.minSdkVersion
                   ^ None of the following functions can be called with the arguments supplied: 
                       public abstract fun minSdkVersion(minSdkVersion: Int): Unit defined in com.android.build.api.dsl.ApplicationDefaultConfig
                       public abstract fun minSdkVersion(minSdkVersion: String?): Unit defined in com.android.build.api.dsl.ApplicationDefaultConfig

  Line 26:         minSdkVersion flutter.minSdkVersion
                   ^ Variable expected

--- a/packages/smooth_app/android/app/build.gradle.kts
+++ b/packages/smooth_app/android/app/build.gradle.kts
@@ -23,7 +23,7 @@ android {
 
     defaultConfig {
         applicationId = "org.openfoodfacts.scanner"
-        minSdk = 23
+        minSdk = flutter.minSdkVersion
         targetSdk = flutter.targetSdkVersion
         versionCode = flutter.versionCode
         versionName = flutter.versionName

